templates/base/Dockerfile

```diff
@@ -1,17 +1,17 @@
-FROM --platform=${TARGETPLATFORM} ubuntu:jammy AS ubuntu-jammy
+FROM --platform=${TARGETPLATFORM} ubuntu:noble AS ubuntu-noble
 
 COPY resources/etc/created-date /etc/created-date
 
 RUN apt-get update && \
-	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes bash ca-certificates curl jq less libnss3 telnet tini tree unzip && \
+	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes adduser bash ca-certificates curl jq less libnss3 telnet tini tree unzip && \
```

Comment:
@Wykenz, do we really need to include the `adduser` command here? Isn't it installed in Ubuntu Noble by default?

Reply:
It is not a default package somehow. I added it because it gave me a "command not found" error